source/release-notes/backend.txt

```diff
@@ -13,6 +13,18 @@ Atlas App Services Changelog
 2022 Releases
 -------------
 
+.. _backend_20221214:
+
+14 December 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: waiting on link for Admin API endpoints for changing expiration times. will add when ready. 
+
+- Raise :ref:`Atlas Function request timeout <function-constraints>`
+  from 150 seconds to 180 seconds.
+- Let users configure expiration times for refresh tokens from the Admin API.
+- Let users create Apps in AWS region Ohio (us-east-2) from the UI.
```

Comment:
this is in prev release notes, maybe we just want to move it up to be accurate?

Reply:
in prev release notes, it was enabled on backend. but this time in the UI as well.